src/style.css

```diff
@@ -1 +1,79 @@
 /* Styles go here */
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  background: #e5e5e5;
+  height: 100%;
+}
+
+.header {
+  --height-heder: 60px;
+  --line-height: 14px;
+  display: flex;
+  justify-content: space-between;
+  padding-right: 50px;
+  height: var(--height-heder);
+  max-width: 1200px;
+  min-width: 1024px;
```

Comment:
delete min and max width

Reply:
if I delete, then the whole layout is spoiled